vcell-cli/src/test/java/org/vcell/cli/testsupport/TestOmexTestReport.java

```diff
@@ -0,0 +1,33 @@
+package org.vcell.cli.testsupport;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.vcell.sedml.testsupport.*;
+
+public class TestOmexTestReport {
```

Comment:
might as well stick in expected results of some kind - or expectThrow if row,column out of range ... or something.

Also, this needs a test category, like:
@Tag("Fast")

Reply:
Had the result checked to see if it matched the expected.